packages/cli-kit/src/private/common/api/graphql.ts

```diff
@@ -0,0 +1,76 @@
+import {buildHeaders, RequestClientError, sanitizedHeadersOutput} from './headers.js'
```

Comment:
All the modules under `private/common/api`  in this PR have transitive modules that depend on Node. Therefore I'd move `grahql` and `headers` under `private/node/api`. In case of doubt, I'd lean on the `node` side unless the module's content is an obvious runtime-agnostic piece of code. For example the lodash-powered functions.